### ISSUE TITLE
Display is not normal after using eclipse calculation tool

### ISSUE THREAD

**Issue by mike805user** [Stellarium/stellarium#2464]
### The display doesn't return to normal after using the Calculations window for eclipses.
Describe or maybe attach a screenshot?
Both images are for the same date and time.  
As it appears after eclipse calculator.
![Screen After Eclipse Display](https://user-images.githubusercontent.com/106281797/170375435-5ab89c86-7471-4131-8153-f074837f64c0.JPG)

As it should appear:
![Normal Dsiplay 2 - same time today](https://user-images.githubusercontent.com/106281797/170375453-ed305a1a-b5bb-4f56-b94f-2c34da88437e.JPG)

### Start Stellarium in full-screen mode.
Open Astronomical Calculations window.
Select Eclipses tab.
Click calculate solar eclipses button.
Select the solar eclipse dated 2024-04-08 and double-click it.  (Any solar or lunar eclipse will show this problem.)
The display centers on the sun, foreground darkens.  Time display on screen confirms the date, just before noon Pacific time.
Click away from the sun to release the focus.
Close the Astronomical Calculations window.
Use "set time to now" button.  The time updates to the correct current time, but the display does not update correctly.  The 
sun position changes from where it was at eclipse time, but doesn't appear to be correct.  Foreground remains very dark.  Sun 
appears to be positioned for late afternoon in the sky.

The only way I know to reset the display is to close Stellarium, then restart it.  A cold restart of the desktop does not correct the problem.

### System
* Stellarium version:  0.22.1, installed 5/25/2022.  Downloaded installable file? stellarium-0.22.1-win64
* Operating system: Windows 7 Home Premium SP1 64bit on Dell XPS 8300 desktop.
* Graphics Card: NVidia GeForce GTX 1060 6GB, using NVidia drivers V472.12.
* Screen type (if applicable): NO scaling or high Dpi. resolution is 1920 x 1200.  Dell 2412M.

### Logfile 
[log.txt](https://github.com/Stellarium/stellarium/files/8774791/log.txt)

logfile `log.txt` is attached

I'm a novice user of Stellarium, which is a fascinating tool.  I don't know if this is a bug or I'm doing something wrong.  I've attached two screen grabs - one right after the eclipse calculation use, the other as it should appear.

Is there something else I'm supposed to do to return the display to reflect the current date and time?

Thanks.

**Comment by mike805user** [comment]
Hi,

 

Thanks for letting me know.  Glad to help.

 

Mike

**Comment by mike805user** [comment]
Hi,

The eclipse lighting problem has been corrected, so thank you for that.  However, I did notice that some of the information pertinent to the eclipse continues to display in the info strip just above the main tool bar.  I've attached a screenshot with the info highlighted in the lower left hand corner.

This is the screen after the eclipse display has been ended, and Set Time to Now button has been used.
![#3 After Eclipse Display - Now](https://user-images.githubusercontent.com/106281797/170739370-fffe3c4d-d480-49e1-8dbc-39ca3052e456.JPG)

Thanks,
Mike
